android/src/main/java/expo/modules/liveupdates/LiveUpdatesHelpers.kt

```diff
@@ -32,12 +31,10 @@ fun getChannelName(context: Context): String {
   return getMetadataFromManifest(context, CHANNEL_NAME_KEY)
 }
 
-fun checkPostNotificationPermission(context: Context) {
-  if (
-    Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU &&
-      ContextCompat.checkSelfPermission(context, Manifest.permission.POST_NOTIFICATIONS) !=
-        PackageManager.PERMISSION_GRANTED
-  ) {
-    throw CodedException("${Manifest.permission.POST_NOTIFICATIONS} permission is not granted.")
+fun checkPostNotificationPermission(context: Context): Boolean {
+  if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+    return ContextCompat.checkSelfPermission(context, Manifest.permission.POST_NOTIFICATIONS) ==
+      PackageManager.PERMISSION_GRANTED
   }
+  return true
 }
```

Comment:
This logic can be compressed into a single boolean expression; we can then use an expression body for the function.